src/index.js

```diff
@@ -854,16 +854,33 @@ function drawKpis(totals, totalsDiff) {
   setKpiDiff('active', (totalsDiff.confirmed - totalsDiff.recovered) - totalsDiff.deceased)
 }
 
-
+/**
+ * @param {string} lastUpdated - MMM DD YYYY, HH:mm JST (e.g. Mar 29 2020, 15:53 JST)
+ */
 function drawLastUpdated(lastUpdated) {
   // Draw the last updated time
 
+  const display = document.getElementById('last-updated');
+
```

Comment:
Could you check here whether display is undefined before continuing?

Reply:
That element is in the static markup: https://github.com/reustle/covid19japan/blob/master/src/index.html#L70

Is there some reason it would be removed dynamically?

Reply:
We're seeing errors consistently where elements we thought should be there don't exist (maybe the DOM hadn't finished loading yet).

I'm actively trying to figure out why that is, but meanwhile, we just try to guard things as much as possible. Thanks!